plugins/openwhisk/src/main/java/com/navercorp/pinpoint/plugin/openwhisk/interceptor/TransactionMetadataInterceptor.java

```diff
@@ -0,0 +1,104 @@
+package com.navercorp.pinpoint.plugin.openwhisk.interceptor;
+
+import com.navercorp.pinpoint.bootstrap.async.AsyncContextAccessor;
+import com.navercorp.pinpoint.bootstrap.context.*;
+import com.navercorp.pinpoint.bootstrap.interceptor.AroundInterceptor;
+import com.navercorp.pinpoint.bootstrap.logging.PLogger;
+import com.navercorp.pinpoint.bootstrap.logging.PLoggerFactory;
+import com.navercorp.pinpoint.bootstrap.util.NumberUtils;
+import com.navercorp.pinpoint.plugin.openwhisk.OpenwhiskConstants;
+import com.navercorp.pinpoint.plugin.openwhisk.descriptor.DefaultMethodDescriptor;
+
+/**
+ * @author Seonghyun Oh
+ */
+public class TransactionMetadataInterceptor implements AroundInterceptor {
+
+    private final PLogger logger = PLoggerFactory.getLogger(TransactionMetadataInterceptor.class);
+
+    private static final DefaultMethodDescriptor TRANSACTION_METHOD_DESCRIPTOR = new DefaultMethodDescriptor("TRANSACTION");
+    private static final DefaultMethodDescriptor TRANSACTION_START_METHOD_DESCRIPTOR = new DefaultMethodDescriptor("TRANSACTION START");
```

Comment:
TRANSACTION_START It would be better if you let me know that this part is Openwhisk.
I suggest that it looks good on the user side because it looks similar to other plugins.
(ex Akka Http Server,  Vert.x HTTP Server, Entry Point Process, RabbitMQ Consumer Invocation)